Dockerfile.e2e

```diff
@@ -43,8 +43,11 @@ FROM base AS builder
 ARG DOCKER_GITCOMMIT
 ENV DOCKER_GITCOMMIT=${DOCKER_GITCOMMIT:-undefined}
 COPY . .
+# Copy test sources tests that use assert can print errors
+RUN mkdir -p /build/${PWD} && find integration integration-cli -name \*_test.go -exec cp --parents '{}' /build${PWD} \;
```

Comment:
I'm using $PWD here as it points to /go/src/github.com/docker/docker and this is where binaries expect the files to be in. This is also why the last COPY statement (below) has destination as /